Source/Domain/Kysect.Tamgly.Core/Aggregates/WorkItemManager.cs

```diff
@@ -79,6 +81,20 @@ public IReadOnlyCollection<WorkItem> GetWorkItems()
             .ToList();
     }
 
+    public IReadOnlyCollection<WorkItem> GetWorkItemWithWrongEstimates()
```

Comment:
```suggestion
    public IReadOnlyCollection<WorkItem> GetWorkItemsWithWrongEstimates()
```
Не смотрел, есть ли где вызовы этого метода, будьте осторожны